build/release/teamcity-mark-build.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
```

Comment:
I updated this script to defined a function, to be in line with other helpers scripts, and added a comment on the function.

Reply:
Ah! I was thinking the script would only do this one thing and a function didn't come to mind for me because it is more than is needed and I was not thinking the script would do multiple things. This works too.